modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Options/Options.es.js

```diff
@@ -197,6 +197,9 @@ const Options = ({
 				id
 			);
 		}
+		else if (property === 'value') {
+			value = normalizeValue(fields, value, fields[index]);
+		}
```

Comment:
@carolmariaabb, please re-use the previous `if` for optimization, something like:
```
if (property === 'value') {
    if (generateKeyword) {
        ...
    }
    else {
        ...
    }
}
```